main.go

```diff
@@ -20,7 +20,10 @@ func main() {
 				printList()
 				return nil
 			}
-			events, err := prepareEventsToTrace(c.StringSlice("event"))
+			if len(c.StringSlice("event")) > 0 && len(c.StringSlice("exclude")) > 0 {
```

Comment:
probably clearer to replace `len(c.StringSlice("flag")) > 0` with `c.IsSet("flag")`